src/index.js

```diff
@@ -272,6 +326,8 @@ class Select extends Component {
           autoFocus
           ref={c => this.searchInput = c}
           onChange={this.handleSearchInputChange}
+          onFocus={this.handleSeachInputFocus}
+          onBlur={this.handleSeachInputBlur}
         />
```

Comment:
สลับ logic นิดนึงน่าจะเข้าใจกว่านะครับใส่ ! แล้วอ่านยากนิดนึง
if (this.shouldRenderSearchInput) {
    return render ปกติ
}
return render อีกแบบนึง